### PR TITLE
Create oyepriyansh.json

### PR DESCRIPTION
**Pull request by oyepriyansh**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5623"></a>

### CODE REVIEW

File: data/oyepriyansh.json [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+		{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/oyepriyansh"
+		},
+
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/oyepriyansh"
+		},
```
Comment (kumarsonsoff3) [inline data/oyepriyansh.json:26]: Please remove this trailing comma `,` here 
```suggestion
		}
```

File: data/oyepriyansh.json [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+		{
+
+			"group": "Socials",
+
+			"name": "Discord: Let`s hang out",
+
+			"url": "https://discord.com/invite/AeAjegXn6D",
+
+			"icon": "FaDiscord"
```
Comment (kumarsonsoff3) [inline data/oyepriyansh.json:146]: There is a missing comma `,`

```suggestion
			"icon": "FaDiscord",
```

File: data/oyepriyansh.json [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+		{
+
+			"group": "Sponsor",
+
+			"name": "Github Sponsor",
+
+			"url": "https://github.com/sponsors/oyepriyansh",
+
+			"icon": "SiGithubsponsors"
```
Comment (kumarsonsoff3) [inline data/oyepriyansh.json:160]: Another missing comma `,` here. 
```suggestion
			"icon": "SiGithubsponsors",
```

File: data/oyepriyansh.json [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+		{
+
+			"group": "Sponsor",
+
+			"name": "Buy me a coffee",
+
+			"url": "https://buymeacoffee.com/oyepriyansh",
+
+			"icon": "SiBuymeacoffee"
```
Comment (kumarsonsoff3) [inline data/oyepriyansh.json:174]:
```suggestion
			"icon": "SiBuymeacoffee",
```

File: data/oyepriyansh.json [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+
+		{
+
+			"group": "Sponsor",
+
+			"name": "Ko-Fi",
+
+			"url": "https://ko-fi.com/oyepriyansh",
+
+			"icon": "SiKofi"
```
Comment (kumarsonsoff3) [inline data/oyepriyansh.json:188]:
```suggestion
			"icon": "SiKofi",
```

File: data/oyepriyansh.json [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+
+		{
+
+			"group": "Sponsor",
+
+			"name": "UPI",
+
+			"url": "https://oyepriyansh.pages.dev/upi",
+
+			"icon": "SiPhonepe"
```
Comment (kumarsonsoff3) [inline data/oyepriyansh.json:202]:
```suggestion
			"icon": "SiPhonepe",
```

File: data/oyepriyansh.json [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+
+		{
+
+			"group": "Sponsor",
+
+			"name": "GiftApp",
+
+			"url": "https://giftapp.com/priyansh",
+
+			"icon": "FaGift"
```
Comment (kumarsonsoff3) [inline data/oyepriyansh.json:216]:
```suggestion
			"icon": "FaGift",
```

File: data/oyepriyansh.json [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+
+		{
+
+			"group": "Sponsor",
+
+			"name": "PayPal",
+
+			"url": "https://paypal.me/oyepriyansh",
+
+			"icon": "FaPaypal"
```
Comment (kumarsonsoff3) [inline data/oyepriyansh.json:230]:
```suggestion
			"icon": "FaPaypal",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey @oyepriyansh, please remove the trailing commas & add missing commas `,` as suggested below to validate your `json`.

**Comment by oyepriyansh** [comment]
i will repon after fixing all errors

**Pull request closed without merging**